code/__HELPERS/mobs.dm

```diff
@@ -263,8 +263,20 @@ This is always put in the attack log.
 	if(istype(target))
 		target.create_attack_log("<font color='orange'>Attacked by [user_str]: [what_done]</font>")
 	log_attack(user_str, target_str, what_done)
-	if(admin_notify)
-		msg_admin_attack("[key_name_admin(user)] vs [key_name_admin(target)]: [what_done]")
+
+	var/loglevel = ATKLOG_MOST
+
+	if(custom_level)

```

Comment:
This logging level system is confusing, arbitrary, and overall not really very great. 
I'd prefer it if you replaced admin_notify with #defines to indicate the desired behavior of the notifications with common behaviors -- 
Something like `ATKLOG_IF_BOTH_HAVE_CKEYS` or `ATKLOG_IF_TARGET_SSD`.

Reply:
The levels aren't reducible to single conditions like that. For example, ATKLOG_FEW means "only the most important stuff: attacks on SSDs, things involving explosives, things involving the engine, gibbing, AI wiping, forcefeeding, acid sprays, and organ extraction". Any time you change your setting, you are told what the setting means (via a chat message). So, it should be quite transparent what they mean.

Reply:
`FEW` is a vague descriptor
Perhaps you could have various "sub-tags" in which these various events would be logged, such as "explosions", "item destruction", etc. - and these could very well be strings, instead of numbers.
And then have a "Critical" logging level which just scans for attack actions within that set of "the most important stuff", doing this mask either by checking a list that contains these "critical" tags (`tag in critical_list` or something) or by doing a bitmask (though the latter approach wouldn't scale effectively for all the tags needed)